### ISSUE TITLE
Newly Added Worlds cause Errors in mysql

### ISSUE THREAD

**Issue by LlmDl** [LlmDl/Towny#2617]
This ticket template is for bug/error reports. If you are suggesting a feature and not reporting a bug please delete all the text here and begin your ticket Title with Suggestion:

### What steps will reproduce the problem?
1. Make a new world and /ta reload
2. Use mysql
3. Get the following:
```
[21:30:08] [Craft Scheduler Thread - 17/WARN]: [Towny] Error: SQL: Field 'plotManagementRevertSpeed' doesn't have a default value --> com.mysql.jdbc.JDBC42PreparedStatement@22f1cde2: REPLACE INTO TOWNY_WORLDS (`plotManagementWildRegenSpeed`, `unclaimedZoneItemUse`, `usingPlotManagementMayorDelete`, `plotManagementIgnoreIds`, `unclaimedZoneSwitch`, `plotManagementMayorDelete`, `forceexplosions`, `forcetownmobs`, `usingPlotManagementWildRegen`, `claimable`, `explosions`, `unclaimedZoneName`, `disableplayertrample`, `firespread`, `forcepvp`, `disablecreaturetrample`, `unclaimedZoneBuild`, `usingPlotManagementDelete`, `pvp`, `usingTowny`, `usingPlotManagementRevert`, `towns`, `unclaimedZoneIgnoreIds`, `name`, `plotManagementDeleteIds`, `PlotManagementWildRegenEntities`, `worldmobs`, `forcefirespread`, `endermanprotect`, `unclaimedZoneDestroy`) VALUES ('20','1','1','GOLD_ORE#LAPIS_ORE#LAPIS_BLOCK#GOLD_BLOCK#IRON_BLOCK#MOSSY_COBBLESTONE#TORCH#MOB_SPAWNER#DIAMOND_ORE#DIAMOND_BLOCK#SIGN_POST#WALL_SIGN#GLOWSTONE','1','WALL_SIGN#SIGN_POST','0','0','1','1','0','Wilderness','1','1','0','1','1','1','0','1','0','','SAPLING#GOLD_ORE#IRON_ORE#COAL_ORE#LOG#LEAVES#LAPIS_ORE#LONG_GRASS#YELLOW_FLOWER#RED_ROSE#BROWN_MUSHROOM#RED_MUSHROOM#TORCH#DIAMOND_ORE#LADDER#RAILS#REDSTONE_ORE#GLOWING_REDSTONE_ORE#CACTUS#CLAY#SUGAR_CANE_BLOCK#PUMPKIN#GLOWSTONE','test-a','BED_BLOCK#TORCH#REDSTONE_WIRE#SIGN_POST#WOODEN_DOOR#WALL_SIGN#STONE_PLATE#IRON_DOOR_BLOCK#WOOD_PLATE#REDSTONE_TORCH_OFF#REDSTONE_TORCH_ON#DIODE_BLOCK_OFF#DIODE_BLOCK_ON','creeper#endercrystal#enderdragon#fireball#smallfireball#largefireball#tntprimed#explosiveminecart','0','0','1','1')
```

### What is your Towny version?
0.91.4.7

**Comment by LlmDl** [comment]
https://pastebin.com/Mw5NVnC4

**Comment by LlmDl** [comment]
Fixed in 0.91.4.8
